docs/security.rst

```diff
@@ -0,0 +1,459 @@
+Report a security issue
+=======================
+
+We take security very seriously.
+Your input and feedback on our security is always appreciated.
+
+You can send urgent or sensitive reports directly to security@twistedmatrix.com
+
+You may use our public key (below) to keep your message safe.
+Provide us with a secure way to respond.
+We'll get back to you as soon as we can.
+
+Twisted is an all volunteer project and there may be some delay before we can respond.
+
+Feel free to follow up if you think an unreasonable amount of time has elapsed without a response!
+
+All releases, including security releases, are announced on the mailing list.
+Consider subscribing to receive notification about the availability of a security fix.
+
+
+Security Procedure for Developers
+=================================
+
+The goal of the normal Twisted development procedure is to make all steps transparent and record all information at all times in a public location - either the issue tracker or a branch.
+
+The goal of the security variation of the Twisted development procedure is to keep track of progress resolving security issues while minimizing the window of time where information useful to attackers is available before a fix for the issue is available to Twisted users.
+
+This process is intended as a helpful recommendation.
+Elements of it may be followed more or less strictly depending on the severity of the issue in question:
+
+#. Begin by filing a ticket which does not describe the issue and simply says 'security issue, description pending' and has the 'security' keyword.
```

Comment:
Happily, I think that this will be unnecessary once we've migrated all the tickets.